xdsl/transforms/mpi_to_llvm.py

```diff
@@ -0,0 +1,525 @@
+import dataclasses
+from abc import ABC
+from typing import TypeVar, cast
+
+from xdsl.dialects.builtin import Signedness, IntegerType
+from xdsl.ir import Operation, SSAValue, OpResult, Attribute, MLContext
+
+from xdsl.pattern_rewriter import RewritePattern, PatternRewriter, op_type_rewrite_pattern, PatternRewriteWalker, \
```

Comment:
If you put the imports in parentheses (what's right of `import`), then it should break at 80 characters automatically.

Reply:
Done!